### DISCUSSION TITLE
Improve auto-save

### DISCUSSION THREAD

**Discussion by eolmapper** [OpenOrienteering/mapper#1946] [Ideas]
For OOM apps in Android:
Detects when the device screen is off. And use this to décide when it start automatic saving without disrupting the work.

**Comment by ollesmaps** [comment]
Do you mean recording of GNSS - gpx?

**Reply by eolmapper** [reply]
Sorry, I was talking about auto-saving as it was indicated in the title. 
I  corrected.
